tests/Behat/MinkContext.php

```diff
@@ -85,6 +103,38 @@ public function assertActiveLanguageSwitcherLink($label): void {
     $this->assertSession()->elementTextContains('css', $selector, $label);
   }
 
+  /**
+   * Asserts that the breadcrumb contains a certain number of elements.
+   *
+   * @codingStandardsIgnoreStart
+   * | element   | text    |
+   * | a         | page    |
+   * | ...       | ...     |
+   * @codingStandardsIgnoreEnd
+   *
+   * @param \Behat\Gherkin\Node\TableNode $table
+   *   The pages data.
+   *
+   * @throws \Behat\Mink\Exception\ElementNotFoundException
+   * @throws \Behat\Mink\Exception\ExpectationException
+   *
+   * @Then the breadcrumb should contain the following item(s):
+   */
+  public function checkBreadcrumbItems(TableNode $table): void {
+    $selector = 'ol.ecl-breadcrumb__segments-wrapper li.ecl-breadcrumb__segment';
+    $breadcrumb = $this->getBreadcrumb();
+    $this->assertSession()->elementsCount('css', $selector, count($table->getHash()), $breadcrumb);
```

Comment:
In the future we will have more breadcrumb elements than two. Let's predict this.